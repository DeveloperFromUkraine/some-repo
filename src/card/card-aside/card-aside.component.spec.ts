import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardAsideComponent } from "./index";
import { ComponentTest } from '../../../test/test-bed/component';

describe('Card Aside', () => {
    let fixture: ComponentFixture<CardAsideComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardAsideComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardAsideComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
}); 