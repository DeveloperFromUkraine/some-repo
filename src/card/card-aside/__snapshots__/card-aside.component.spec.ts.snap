// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Aside should match snapshot 1`] = `
<ign-card-aside>
  
</ign-card-aside>
`;
